src/helpers/fallbackauthdialog.ts

```diff
@@ -65,3 +97,62 @@ function handleResponse(response) {
     Office.context.ui.messageParent(JSON.stringify({ status: "success", result: response.accessToken }));
```

Comment:
In my code I also send the home account id which you can use to access the cache as MSAL can cache multiple client accounts. (so for multiple REST calls you don't have to keep popping up the fallback dialog).
`JSON.stringify({ status: "success", result: response.accessToken, accountId: response.account.homeAccountId }),`

Reply:
Thanks.  I got part of that but missed this piece of it.  Any suggestions on making sure it works?

Reply:
You have to force fallback auth to occur, then keep clicking the button to call the API (and Graph). You shouldn't see the popup auth dialog again if the cache is working correctly.

Reply:
There doesn't seem to be any accounts to compare to the next time around.  Made some changes that should handle it if it's there and skip using it if there aren't any accounts to find.  Will have to investigate more later.